src/ProxyEvent.ts

```diff
@@ -56,20 +57,27 @@ export interface ProxyEvent {
         accountId: string,
```

Comment:
I think `context` needs to be renamed to `requestContext` as per https://github.com/Giftbit/cassava/issues/4 both here and `RouterEvent`.  Then `Router.proxyEventToRouterEvent` will need to track that change.

Reply:
Updated.